### ISSUE TITLE
[Noto Sans Mono] `zero` and `liga` features

### ISSUE THREAD

**Issue by twardoch** [googlei18n/noto-fonts#831]
1. In **NotoSansMono-MM.glyphs**, there is the `zero` feature which performs `sub zero by zero.slash;`, but the font contains a glyphs `zero` which has a slash, and a glyph `zero.slash` which... also has the slash (in fact, it's just a composite of `zero`). 

I think having a non-slashed zero would be useful as an alternate. I propose that the glyph `zero.slash` is renamed to `zero.noslash`, and the slash element is removed from it. Then, the `zero` feature should be rewritten to say `sub zero by zero.noslash;` (so in fact, it actually will replace the slashed zero with the unslashed one), and the feature `ss01` should be added to perform the same behavior (the `zero` feature is not acessible in all apps). 

2. The ligatures such as `f_i` and `f_l` are nice, but it’s confusing to have them on by default in a monospace font. I propose that the feature `liga` gets replaced by `dlig`, with the same contents, so we have 

```
feature dlig { 
  sub f f l by f_f_l;
  sub f f i by f_f_i;
  sub f i by f_i;
  sub f l by f_l;
  sub f f by f_f;
} dlig;
```

In addition, the feature `hlig` (historical ligatures) should be added: 

```
feature hlig { 
  sub longs t by longs_t;
  sub s t by s_t;
} hlig;
```

This way, Noto Sans Mono by default will have no ligatures by default (no `liga` feature), but the ligatures will still be accessible via opt-in.

**Comment by moyogo** [comment]
> Then, the zero feature should be rewritten to say sub zero by zero.noslash; (so in fact, it actually will replace the slashed zero with the unslashed one)

@twardoch I don’t think that makes sense, at least not with how the 'zero' feature is currently described in the specs. https://www.microsoft.com/typography/otspec/features_uz.htm#zero says “This feature allows the user to change from the default 0 to a slashed form”.

**Comment by twardoch** [comment]
OK, in this case, the `zero` feature right now doesn’t do anything significant, so it should probably be removed. However, it would still be useful to add a `zero.noslash` glyph and implement the `zero -> zero.noslash` substitution in some other feature (e.g. `ss01`).

**Comment by brawer** [comment]
Somewhat related, the sequence `U+0030 U+FE00` should be slashed as per [Unicode Standardized Variants](http://unicode.org/Public/UCD/latest/ucd/StandardizedVariants.txt).
